backend/models/dropoffLogSchema.js

```diff
@@ -11,9 +11,8 @@ const DropoffLogSchema = new mongoose.Schema(
     name: String,
     recipientEntityType: String,
     demographic: String,
-    foodType: String,
     area: String,
-    lbsDroppedOff: Number,
+    foodAllocation: {String, Number},
```

Comment:
Can you change this to be a Map data type 
https://mongoosejs.com/docs/schematypes.html#maps